src/feature_collection.rs

```diff
@@ -33,7 +33,7 @@ use ::{Bbox, Crs, Error, Feature, FromObject, util};
 ///
 /// Serialization:
 ///
-/// ```rust
+/// ```norun
```

Comment:
It was just the `extern crate rustc_serialize`. I don't know why I ignored the test instead of just remove the crate import... It's not needed. It's fixed now.